README.md

```diff
@@ -351,152 +352,185 @@ or equal to those of the children and the lowest key is in the root node
-│   ├── balancedBinaryTree.java
-│   ├── battleshipsInABoard.java
-│   ├── convertSortedArrayToBinarySearchTree.java
-│   ├── maximumDepthOfABinaryTree.java
-│   ├── numberOfIslands.java
-│   ├── populatingNextRightPointersInEachNode.java
-│   └── sameTree.java
-├── Design
-│   └── zigzagIterator.java
-├── DivideAndConquer
-│   ├── expressionAddOperators.java
-│   └── kthLargestElementInAnArray.java
-├── DynamicProgramming
-│   ├── bombEnemy.java
-│   ├── climbingStairs.java
-│   ├── combinationSumIV.java
-│   ├── countingBits.java
-│   ├── editDistance.java
-│   ├── houseRobber.java
-│   ├── paintFence.java
-│   ├── paintHouseII.java
-│   ├── regularExpressionMatching.java
-│   ├── sentenceScreenFitting.java
-│   ├── uniqueBinarySearchTrees.java
-│   └── wordBreak.java
-├── HashTable
-│   ├── binaryTreeVerticalOrderTraversal.java
-│   ├── findTheDifference.java
-│   ├── groupAnagrams.java
-│   ├── groupShiftedStrings.java
-│   ├── islandPerimeter.java
-│   ├── loggerRateLimiter.java
-│   ├── maximumSizeSubarraySumEqualsK.java
-│   ├── minimumWindowSubstring.java
-│   ├── sparseMatrixMultiplication.java
-│   ├── strobogrammaticNumber.java
-│   ├── twoSum.java
-│   └── uniqueWordAbbreviation.java
-├── LinkedList
-│   ├── addTwoNumbers.java
-│   ├── deleteNodeInALinkedList.java
-│   ├── mergeKSortedLists.java
-│   ├── palindromeLinkedList.java
-│   ├── plusOneLinkedList.java
-│   ├── README.md
-│   └── reverseLinkedList.java
-├── Queue
-│   └── movingAverageFromDataStream.java
-├── README.md
-├── Sort
-│   ├── meetingRoomsII.java
-│   └── meetingRooms.java
-├── Stack
-│   ├── binarySearchTreeIterator.java
-│   ├── decodeString.java
-│   ├── flattenNestedListIterator.java
-│   └── trappingRainWater.java
-├── String
-│   ├── addBinary.java
-│   ├── countAndSay.java
-│   ├── decodeWays.java
-│   ├── editDistance.java
-│   ├── integerToEnglishWords.java
-│   ├── longestPalindrome.java
-│   ├── longestSubstringWithAtMostKDistinctCharacters.java
-│   ├── minimumWindowSubstring.java
-│   ├── multiplyString.java
-│   ├── oneEditDistance.java
-│   ├── palindromePermutation.java
-│   ├── README.md
-│   ├── reverseVowelsOfAString.java
-│   ├── romanToInteger.java
-│   ├── validPalindrome.java
-│   └── validParentheses.java
-├── Tree
-│   ├── binaryTreeMaximumPathSum.java
-│   ├── binaryTreePaths.java
-│   ├── inorderSuccessorInBST.java
-│   ├── invertBinaryTree.java
-│   ├── lowestCommonAncestorOfABinaryTree.java
-│   ├── sumOfLeftLeaves.java
-│   └── validateBinarySearchTree.java
-├── Trie
-│   ├── addAndSearchWordDataStructureDesign.java
-│   ├── implementTrie.java
-│   └── wordSquares.java
-└── TwoPointers
-    ├── 3Sum.java
-    ├── 3SumSmaller.java
-    ├── mergeSortedArray.java
-    ├── minimumSizeSubarraySum.java
-    ├── moveZeros.java
-    ├── removeDuplicatesFromSortedArray.java
-    ├── reverseString.java
-    └── sortColors.java
-
-18 directories, 124 files
-```
+## Directory of populer Algorithms and Data structures Implementation in Java. 
```

Comment:
I don't think what you added is correct (and it also has misspellings and grammatical errors). The reason why it's not correct is because these are not implementations of popular algorithms and data structures, they are solutions to programming challenges.